Sources/FirebaseFirestore/DocumentReference+Swift.swift

```diff
@@ -120,8 +120,9 @@ extension DocumentReference {
   }
 }
 
-extension DocumentReference: CustomDebugStringConvertible {
-  public var debugDescription: String {
-    String(self.ToString())
-  }
-}
+// Comment out to work around WIN-971
+// extension DocumentReference: CustomDebugStringConvertible {
+//   public var debugDescription: String {
```

Comment:
Does the example project still build after this change? I know we're using `debugDescription` on some of the objects to print to the console. As an aside, other objects in this SDK have `debugDescription` implemented in a similar fashion, why does only this one need to be commented out?

Reply:
Good question. I don't know the answer. Arc builds. How do I build the example project?

Reply:
You should be able to use `swift build --product FirebaseUI`

Reply:
This part of the change turned out unnecessary :) Resolving the conversation.